scripts/sommelier

```diff
@@ -289,6 +289,9 @@ if [[ ! -d "$WINE_PLATFORM" ]]; then
   fi
 fi
 
+# Fix for nvidia drivers on Ubuntu 21.10 and other very modern systems (part 1 of 2)
```

Comment:
Could you make this comment a bit more prescriptive? Specifically, explain you remove `$SNAP_LIBRARY_PATH` here in order to append it later in part 2, because it needs to be the last path, not the first in order to solve issues with the nvidia driver on 21.10+ systems.